addons/point_of_sale/models/pos_order.py

```diff
@@ -574,6 +585,10 @@ def _create_invoice(self, move_vals):
                 })
         return new_move
 
+    def get_amount_unpaid(self):
+        self.ensure_one()
+        return self.currency_id.round(self._get_rounded_amount(self.amount_total) - self.amount_paid)
+
```

Comment:
move this to pos_online_payment, unless you can find a place here in point_of_sale to call it.